schema.sql

```diff
@@ -14,3 +14,27 @@ CREATE TABLE animals (
 -- Day 2
 ALTER TABLE animals
 ADD species VARCHAR(100);
+
+-- Day 3
+CREATE TABLE OWNERS (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  full_name VARCHAR(250) NOT NULL,
+  age INT NOT NULL
+);
+
+CREATE TABLE species (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
```

Comment:
- [ ] Kindly change the data type for `id` as proposed for the `owners` table above.